src/cart/crt_iv.c

```diff
@@ -1703,27 +1703,38 @@ crt_hdlr_iv_sync_aux(void *arg)
 	switch (sync_type->ivs_event) {
 	case CRT_IV_SYNC_EVENT_UPDATE:
 	{
-		d_sg_list_t tmp_iv;
+		d_sg_list_t	tmp_iv;
+		d_iov_t		*tmp_iovs;
 
 		rc = iv_ops->ivo_on_get(ivns_internal, &input->ivs_key,
-				0, CRT_IV_PERM_WRITE, &iv_value, &user_priv);
-
-		tmp_iv = iv_value;
+				0, CRT_IV_PERM_READ, &iv_value, &user_priv);
```

Comment:
it probably not a big deal, but it probably be better to keep it as WRITE, since the following callback is refresh, so to give DAOS more information about the accessing intent of the iv entry.

Reply:
as discussed in PM, and as per current model, iv_on_get() with PERM_WRITE indicates that returned iv_value could be written to. PERM_READ indicates that contents will only be read and not modified.
Since this code path does not modify contents of the iv_value, as per current model it should be READ